coding-the-legislation/25_vectorial_computing.md

```diff
@@ -20,6 +20,33 @@ array([45, 42, 17])
 
 Basic operations such as `+` or `*` behave the same way on vectors than on numbers, you can thus use them in OpenFisca. However, some operations and structures must be adapted.
 
+## Scalars
+
+As a first-time developer, it is natural to write such a formula
+```py
+def formula(person):
+    rebate = 610
```

Comment:
I'm afraid this example, while being straightforward, can actually be confusing as it shows that it is okay to return a magic number in a formula. Since this is the doc, it mostly targets people who are unsure about what is right and what isn't in OpenFisca, and I believe it is very important to give consistent, best-practice examples.

What about replacing `610` with some parameter fetch, or find an example where it is seemingly legit to return a scalar value from a simple computation made on a variable fetch (that is, legit until you understand how the value should be different if there were several `person`)? 🙂